src/components/parameters/ParameterBaseSequenceList.svelte

```diff
@@ -0,0 +1,107 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import ParameterName from './ParameterName.svelte';
+  import type { FormParameter, ParameterType } from '../../types/parameter.js';
+  import type { ActionArray } from '../../utilities/useActions.js';
+  import { createEventDispatcher } from 'svelte';
+  import SearchableDropdown from '../ui/SearchableDropdown.svelte';
+  import ParameterUnits from './ParameterUnits.svelte';
+  import ParameterBaseRightAdornments from './ParameterBaseRightAdornments.svelte';
+  import { userSequences } from '../../stores/sequencing';
+  import type { DropdownOptions, SelectedDropdownOptionValue } from '../../types/dropdown';
+
+  export let disabled: boolean = false;
+  export let formParameter: FormParameter;
+  export let hideRightAdornments: boolean = false;
+  export let labelColumnWidth: number = 200;
+  export let level: number = 0;
+  export let levelPadding: number = 20;
+  export let parameterType: ParameterType = 'activity';
+  export let use: ActionArray = [];
+  export let workspaceId: number | null;
```

Comment:
Building off of my previous comment above, not all Parameter components are used within a workspace. It looks like you're using the `workspaceId` to filter out some options. Could this maybe instead just take the list of options that a HOC can provide?